README.md

```diff
@@ -1 +1,167 @@
+
+- **Use Webpack**
+- **Use ES6**
+- **Use Modules**
+- **Use basic page structure**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">JavaScript</a></li>
+  </ul>
+</details>
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- LIVE DEMO -->
+ 
+## 🚀 Live Demo <a name="live-demo"></a>
+ 
+- Coming soon...
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- GETTING STARTED -->
+ 
+## 💻 Getting Started <a name="getting-started"></a>
+ 
+To get a local copy up and running, follow these steps.
+ 
+### Prerequisites
+ 
+In order to run this project you need:
+ 
+- A computer.
+- Internet access.
+- A modern web browser.
+ 
+### Setup
+ 
+Clone this repository to your desired folder:
+ 
+```sh
+  cd my-folder
+  git clone git@github.com:https://github.com/QueenterJuma/-Leaderboard-API.git
+```
+ 
+### Install
+ 
+Install this project with:
+ 
+```sh
+  cd my-folder
+  npm-install
+```
+### Usage
+ 
+To run the project, open index.html in your text editor,right-click and open with live server,it will be opened in your web browser.
+ 
+<!-- AUTHORS -->
+ 
+## 👥 Author <a name="author"></a>
+
+👤 **Queenter Juma**
+
+- GitHub:(https://github.com/QueenterJuma)
+- Twitter: (https://twitter.com/QueenJumah)
+- LinkedIn:(https://www.linkedin.com/in/queenter-juma-54a0b1222/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [x] **Send and receive data from API**
+- [x] **Final touches**
+
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+- [ ] **Homepage**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/QueenterJuma/-Leaderboard-API/issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- SUPPORT -->
+ 
+## ⭐️ Show your support <a name="support"></a>
+ 
+If the project was great as your view, don't hesitate to share and give it a star.
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- ACKNOWLEDGEMENTS -->
+ 
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+ 
+Inspired by the external Leaderboard API service, thanks to them.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- LICENSE -->
+ 
+## 📝 License <a name="license"></a>
+ 
+This project is [MIT](https://github.com/QueenterJuma/-Leaderboard-API/blob/main/LICENCE) licensed.
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- README.md
+Displaying README.md. -->
```

Comment:
- [ ] Kindly ensure that you add the `MIT`. The MIT license gives users :person_curly_hair:  express permission to reuse code for any purpose.
**Key note: ** The MIT(lincense) file has been added, however, the link is broken. Kindly ensure the link is leading to the license when clicked.